sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -1020,6 +1006,10 @@ abstract class CodeGenerator[InType <: AnyRef, OutType <: AnyRef] extends Loggin
 }
 
 object CodeGenerator extends Logging {
+
+  // This is the value of HugeMethodLimit in the OpenJDK JVM settings
+  val DEFAULT_OPENJDK_JVM_HUGE_METHOD_LIMIT = 8000
```

Comment:
I think just `DEFAULT_JVM_HUGE_METHOD_LIMIT` is ok. The comment already explains it is from OpenJDK.

Reply:
ok, I'll update